sdk/core/azure-core/README.md

```diff
@@ -93,6 +93,9 @@ If you would like to become an active contributor to this project please follow
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create new Pull Request
 
+@Random
+@NotRandom
```

Comment:
Should these be inside comments to hide it from the user reading this?

Reply:
Added it to test the change, will remove if it passes.

Reply:
Removing in next commit